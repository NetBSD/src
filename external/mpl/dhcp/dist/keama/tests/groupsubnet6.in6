# Group with DHCPv6 subnet declaration config

# parameter which will be changed in subnet
default-lease-time 1200;

# group declaration
group foobar {
    # option
    option dhcp6.domain-search "example.com", "example.org";

    # parameters
    default-lease-time 3600;

    # DHCPv4 subnet declaration
    subnet6 2001::/64 {
        # at least one pool is required
        pool6 {
            range6 2001::100 2001::200;
        }
	interface "en0";
        default-lease-time 1800;	
	option dhcp6.lq-relay-data 2001::1 "foobar";
    }
}
