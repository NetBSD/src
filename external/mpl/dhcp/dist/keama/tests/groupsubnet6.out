{
  # Group with DHCPv6 subnet declaration config
  # parameter which will be changed in subnet
  "Dhcp6": {
    "valid-lifetime": 1200,
    "interfaces-config": {
      "interfaces": [
        "en0"
      ]
    },
    "subnet6": [
      # DHCPv4 subnet declaration
      {
        "id": 1,
        "subnet": "2001::/64",
        "pools": [
          # at least one pool is required
          {
            "pool": "2001::100 - 2001::200"
          }
        ],
        "interface": "en0",
        "valid-lifetime": 1800,
        "option-data": [
          {
            "space": "dhcp6",
            "name": "lq-relay-data",
            "code": 47,
//          "original-data": "2001::1 \"foobar\"",
            "data": "2001::1, 666f6f626172"
          },
          # option
          {
            "space": "dhcp6",
            "name": "domain-search",
            "code": 24,
//          "original-data": "\"example.com\", \"example.org\"",
            "data": "example.com, example.org"
          }
        ]
      }
    ]
  }
}
